### PR TITLE
Update from 2.4.4 to 2.4.9

### PR DESCRIPTION
**Pull request by cbouss**
Jira:
- https://anaconda.atlassian.net/browse/PKG-41

Changes:
- Set to 2.4.9
- Update about section

Change log: https://github.com/libexpat/libexpat/blob/master/expat/Changes
Changes 2.4.4 -> 4.4.9 : https://github.com/libexpat/libexpat/compare/R_2_4_4...R_2_4_9
https://libexpat.github.io
https://github.com/libexpat/libexpat

CVE fixes included:
- CVE-2022-40674
- CVE-2022-25236
- CVE-2022-25235
- CVE-2022-25313
- CVE-2022-25314
- CVE-2022-25315

Relax expat pinning on aggregate:
https://github.com/AnacondaRecipes/aggregate/pull/235

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
`Coreutils` seems to be a dependency for the package. We might want to verify what tools we should use for `win 64`.

https://github.com/libexpat/libexpat/blob/R_2_4_9/Brewfile#L4

**Comment by cbouss** [comment]
> `Coreutils` seems to be a dependency for the package. We might want to verify what tools we should use for `win 64`.
> 
> https://github.com/libexpat/libexpat/blob/R_2_4_9/Brewfile#L4

As far as I see in the code and understand, coreutils is used in one of their Travis CI script. But I don't see it being required for building.

You can find the Windows build options here:
https://github.com/libexpat/libexpat/blob/R_2_4_9/expat/CMakeLists.txt#L127
https://github.com/libexpat/libexpat/blob/R_2_4_9/expat/CMake.README

And the Linux/Mac build options there:
https://github.com/libexpat/libexpat/blob/R_2_4_9/expat/configure.ac
https://github.com/libexpat/libexpat/blob/R_2_4_9/expat/README.md#a-building-with-the-configure-script-ie-gnu-autotools

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,11 +34,12 @@ about:
   home: https://libexpat.github.io
   license: MIT
   license_family: MIT
-  # Note that the file is in a subdirectory expat/COPYING the expat is in a subdirectory
   license_file: COPYING
-  summary: 'Expat XML parser library in C'
+  license_url: https://github.com/libexpat/libexpat/blob/master/expat/COPYING
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:38]: For v2.4.9, it was copied to libexpat/COPYING, see https://github.com/libexpat/libexpat/commit/34f57a0b3033bd6eff689e613ebf3eed56b2d1c7

**Comment by skupr-anaconda** [comment]
> > `Coreutils` seems to be a dependency for the package. We might want to verify what tools we should use for `win 64`.
> > https://github.com/libexpat/libexpat/blob/R_2_4_9/Brewfile#L4
> 
> As far as I see in the code and understand, coreutils is used in one of their Travis CI script. But I don't see it being required for building.

Yep, it's for CI Travis only  https://github.com/libexpat/libexpat/commit/9aaaac076ab3f8ead296c2709956f107e4510959

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml:38]: They placed it at two paths. 
https://github.com/libexpat/libexpat/blob/R_2_4_9/expat/COPYING
https://github.com/libexpat/libexpat/blob/R_2_4_9/COPYING

**Pull request merged**